purchase_request_usage_department/models/hr_department.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
@aheficent Bro It's Not a `AGPL`, Please Rename as per README.

Reply:
Noticed. Thanks bro!